hs_core/models.py

```diff
@@ -1822,30 +1822,113 @@ def check_relations(self, stop_on_error=False, log_errors=True,
                     print("relation {} {} {} (this does not exist)"
                           .format(self.short_id, r.type, target))
 
+    def fix_irods_user_paths(self, log_actions=True, echo_actions=False, return_actions=False):
+        """
+        Move iRODS user paths to the locations specified in settings
+
+        :param log_actions: whether to log actions to Django log
+        :param echo_actions: whether to print actions on stdout
+        :param return_actions: whether to collect actions in an array and return them.
+
+        This is a temporary fix to the user resources, which are currently stored like
+        federated resources but whose paths are dynamically determined. This function points
+        the paths for user-level resources to where they are stored in the current environment,
+        as specified in hydroshare/local_settings.py.
+
+        * This only does something if the environment is not a production environment.
+        * It is idempotent, in the sense that it can be repeated more than once without problems.
+        * It must be done once whenever the django database is reloaded.
+        * It does not check whether the paths exist afterward. This is done by check_irods_files.
+        """
+        logger = logging.getLogger(__name__)
+        actions = []
+        ecount = 0
+
+        # location of the user files in production
+        defaultpath = getattr(settings, 'HS_USER_ZONE_DEFAULT_PATH',
+                              '/hydroshareuserZone/home/localHydroProxy')
```

Comment:
@alvacouch ```HS_USER_ZONE_DEFAULT_PATH``` is not currently in our local_settings.py, I suggest to just assign it directly to  ```/hydroshareuserZone/home/localHydroProxy``` rather than getting it from settings. If you think it is better to add another setting variable for this, I suggest to use HS_PRODUCTION_USER_ZONE_PATH or something like that to explicitly indicate it is production user zone path. Also, if you do ```from django.conf import settings```, you can directly access these variables by doing settings.VARIABLE_NAME which might be a bit simplier.